base/Dockerfile.rhel

```diff
@@ -6,6 +6,7 @@ RUN INSTALL_PKGS=" \
       " && \
     if [ ! -e /usr/bin/yum ]; then ln -s /usr/bin/microdnf /usr/bin/yum; fi && \
     yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
-    yum clean all && rm -rf /var/cache/*
+    yum clean all && rm -rf /var/cache/* && \
+    echo 'skip_missing_names_on_install=0' >> /etc/yum.conf
```

Comment:
Do we need a leading newline?

Reply:
Not that I saw when I tested in the container manually.